src/main/java/org/dasein/cloud/TaskInProgressException.java

```diff
@@ -0,0 +1,39 @@
+package org.dasein.cloud;
+
+import javax.annotation.Nonnegative;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * User: daniellemayne
+ * Date: 16/12/2015
+ * Time: 12:21
+ */
+public class TaskInProgressException extends CloudException {
+    public TaskInProgressException(@Nonnull String msg) {
+        super(msg);
+        this.errorType = CloudErrorType.TASK_IN_PROGRESS;
+    }
+
+    public TaskInProgressException(@Nonnull String msg, @Nonnull Throwable cause) {
+        super(msg, cause);
+        this.errorType = CloudErrorType.TASK_IN_PROGRESS;
+    }
+
+    public TaskInProgressException(@Nonnegative int httpCode, @Nullable String providerCode, @Nonnull String msg) {
```

Comment:
Same here, TaskInProgressException is a concrete enough exception that doesn't need to provide the technical internals like http code and provider code. It would probably unlikely to have a supplemental Throwable neither.